services/ui-src/src/components/cards/TemplateCard.tsx

```diff
@@ -0,0 +1,152 @@
+import { useFlags } from "launchdarkly-react-client-sdk";
+// components
+import { Button, Flex, Heading, Image, Text, Link } from "@chakra-ui/react";
+import { Card, TemplateCardAccordion } from "components";
+// utils
+import { useNavigate } from "react-router-dom";
+import { useBreakpoint } from "utils";
+import { AnyObject } from "types";
+// assets
+import downloadIcon from "assets/icons/icon_download.png";
+import nextIcon from "assets/icons/icon_next_white.png";
+import spreadsheetIcon from "assets/icons/icon_spreadsheet.png";
+
+export const TemplateCard = ({
+  templateName,
+  verbiage,
+  cardprops,
+  isDisabled,
+  ...props
+}: Props) => {
+  const { isDesktop } = useBreakpoint();
+  const navigate = useNavigate();
+
+  const mlrReport = useFlags()?.mlrReport;
```

Comment:
Right now we don't have any LD flags setup, I think we can get rid of this `mlrReport` variable for now.